internal/cmd/kafka/command_link_create.go

```diff
@@ -167,36 +190,129 @@ func (c *linkCommand) create(cmd *cobra.Command, args []string) error {
 	return nil
 }
 
+var printableLinkConfigs = map[string]bool{
```

Comment:
We have `types.Set{}` that might be more user friendly to use here (no need to type `true` a bunch of times)